ckanext/stadtzhtheme/templates/home/snippets/search.html

```diff
@@ -3,14 +3,14 @@
 
   <form class="module-content search-form" method="get" action="{% url_for controller='package', action='search' %}">
     <div class="search-input control-group search-giant">
-      <input id="ogdzh_search"
-             type="text"
-             class="search"
-             name="q"
-             value="{{ query }}"
-             autocomplete="off"
-             placeholder="{{ placeholder }}"
-             data-module="ogdzh_autocomplete">
+      <input id="ogdzh_search" type="text"
```

Comment:
Nitpicky here, but if you touch the code anyways, why not move `type="text"` to its own line as the other parameters already are? :)